packages/components/src/Persona.purs

```diff
@@ -45,6 +45,7 @@ foreign import loginApi :: Api
 foreign import usersApi :: Api
 foreign import rawJSONStringify :: Foreign -> String
 foreign import rawJSONParse :: String -> Foreign
+foreign import constructUserFromObject :: Foreign -> User
```

Comment:
Does this always succeed?

Reply:
It works as long as it uses the same `User` both on the server and the client. I wouldn't use this in the app but I'd expect the web site and Mosaico to stay in close sync. Luckily `User` should be pretty static by now and if we made changes to it we'd need to do it as a multi stage process anyway due to Mitt Konto.

If we wanted to have a better `User` constructor then we should have it in `persona-javascript-client` since I wouldn't want to diverge from what it uses in Mosaico. Or replace other users of its `User` construction as well. This is as good as we have right now.

Reply:
Or we could replace it with some lite version of `User` that only handles the fields that Mosaico uses. Does it even use anything other than just the first name? It's not like we'd ever want to see subscriptions, entitlements are going to be checked other way anyway.